zschema/compounds.py

```diff
@@ -192,8 +192,11 @@ def validate(self, value):
             raise DataValidationException("record is not a dict", str(value))
         for subkey, subvalue in value.items():
             if subkey not in self.definition:
-                raise DataValidationException("%s is not a valid subkey of root",
-                                              subkey)
+                port = Port(subkey)
```

Comment:
`Port` is defined in `zschema`, I'm not sure what you mean by it being specific to Censys.
Example Schema:
```python
ssh = Record({
    Port(22): SubRecord({
        "banner": String(),
    }),
})
```

Example object:
```json
{
    "22": {
        "banner": "OpenSSH-7.2"
    }
}
```

If you attempt to validate this object against that schema without this PR, you will always get "`Error: Unexpected key '22'`"